lib/ansible/modules/files/find.py

```diff
@@ -372,7 +377,8 @@ def main():
             follow=dict(type='bool', default=False),
             get_checksum=dict(type='bool', default=False),
             use_regex=dict(type='bool', default=False),
-            depth=dict(type='int'),
```

Comment:
You cannot remove existing params, they either need to be an alias, or it needs to toggle different functionality.

Reply:
Of course you are right, old playbooks shouldn't be affected of this change. Like i mentioned in my initial post, I was unsure if I should add the new params 'max' and 'min' or if I should only add 'min' and let 'depth' unchanged. 
Seems like while trying out both options I missed something in my code. I pushed a new commit to fix this, and only add 'min_depth'. 

Or would be an alias the better option?

Reply:
an alias would work